types/mathjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "decimal.js": "^10.0.0"
+        "decimal.js": "^10.0.1"
```

Comment:
A little pedantic, but this doesn't actually matter. Since we are using the semver: `^`, we will get all minor and patch versions for free. Usually I only update the listed version whenever a change specifically requires a higher version than is listed.

For instance, if your change doesn't specifically rely on something on version `10.0.1` of decimal.js, then it is actually better to leave the version as unconstrained as possible.

Reply:
You are right, I reverted it.